architecture-decision-records/0025-automatic-code-merge.md

```diff
@@ -0,0 +1,28 @@
+#25. Automatic code merging
+
+**Date:** 2022-03-15
+
+## Context
+We have two services now, dependabot and Scala Steward, which create pull requests for dependency updates. 
```

Comment:
Is it worth linking the part about Scala Steward back to the ADR about moving to GutHub Actions which allows us to use Scala Steward?

Reply:
I've taken that bit out because it talks about forks which we're not using now so I think this is ok.